pkg/autoops/api/api.go

```diff
@@ -836,6 +833,13 @@ func (s *AutoOpsService) ExecuteAutoOps(
 	if err := s.validateExecuteAutoOpsRequest(req, localizer); err != nil {
 		return nil, err
 	}
+	triggered, err := s.checkIfHasAlreadyTriggered(ctx, localizer, req.Id, req.EnvironmentNamespace)
```

Comment:
I had to move this implementation from the transaction to check if it had already been triggered because the response returns the `AlreadyTriggered` boolean.

https://github.com/bucketeer-io/bucketeer/pull/196/files#diff-13112ed94680ecaec33373531ff5c65e48cecae142657203e04d967a185684d5L862-L864